pkg/mmesh/grpc_resolver_test.go

```diff
@@ -142,14 +141,6 @@ func reconcile(t *testing.T, kr *KubeResolver) {
 	assert.Nil(t, err)
 }
 
-func makeKubeResolver(namespace string, client client.Client) *KubeResolver {
-	return &KubeResolver{
```

Comment:
Looks makeKubeResolver is still needed in line 78, kr := makeKubeResolver("namespace", mClient)?

Reply:
I saw it is moved... so ignore previous comment